book/website/ethical-research/activism/activism-env-impact.md

```diff
@@ -0,0 +1,168 @@
+
+Finally some data centers are shifting towards renewable energy sources (solar, wind, hydroelectric power) and reduce their dependencies on fossil fuels.
+
+The greenest data/HPC centers are listed on the [GREEN500 list](https://www.top500.org/lists/green500/).
+
+Whenever you can choose, select data centers that are committed to reduce their carbon footprint.
+
+### Other resources
+#### Computation
+* [GreenvAlgorithms](https://www.green-algorithms.org/) 
+* [Green Algorithms: Quantifying the Carbon Footprint of Computation](https://onlinelibrary.wiley.com/doi/10.1002/advs.202100707) 
+* [Carbon footprint estimation for computational research](https://www.nature.com/articles/s43586-023-00202-5.epdf?sharing_token=QTc-q5_jBfbwbgHrxjAF3NRgN0jAjWel9jnR3ZoTv0M5cDtvWq4NDOHdun0YacmqG8iEJ146ZGUvxIp3izWvG3qqEI0oN4e5AgJh1ioWOFtL_b-yexmwuBL7nKWggSD22xXBBQe-_M_1jL9gC5WNNOYCZkXrOcuiKp7L_6DvyMI%3D)
+* [Energy and Policy Considerations for Deep Learning in NLP](https://www.aclweb.org/anthology/P19-1355) 
+* [Green AI](http://arxiv.org/abs/1907.10597) 
+* [On the Dangers of Stochastic Parrots: Can Language Models Be Too Big?](https://doi.org/10.1145/3442188.3445922) 
+* [Digital Humanities Climate Coalition
+](https://www.cdcs.ed.ac.uk/digital-humanities-climate-coalition)
+* [Wren Carbon calculator](https://www.wren.co/) 
+* [Turner (2022) Study of the impact of CPU frequency on ARCHER2](https://www.archer2.ac.uk/news/2022/12/12/CPUFreq.html). 
+* [ARCHER2 documentation](https://docs.archer2.ac.uk/user-guide/energy/) 
+
+
+#### Data
+* [Green Scientific Data Compression](https://centaur.reading.ac.uk/79584/1/tgsdcthiia18-towards_green_scientific_data_compression_through_high_level_i_o_interfaces.pd)
+* [Environmental Impact of Video streaming](https://research.vu.nl/ws/portalfiles/portal/214278850/Jancovic_Marek_Keilbach_Judith_2023_Streaming_Against_the_Environment._Digital_Infrastructures_Video_Compression_and_the_Environmental_Footprint_of_Video_Streaming.pdf)
+* [Next-generation high-temperature data centers](https://doi.org/10.1016/j.rser.2022.112991)
+* [Zero energy consumption data centers](https://doi.org/10.1016/j.energy.2022.125495)
+
+## Credits
+
+This document draws on discussion from the Software Sustainability Institute’s Collaborations Workshop 2023 (CW23), and in particular on the “Raspberry” discussion and speed blogging session on “How do you track the environmental impact of computing?”. We thank all participants of CW23 for engaging around this topic.  

```

Comment:
It would be good to add some links for this:
```suggestion
This document draws on discussion from the [Software Sustainability Institute](https://www.software.ac.uk/)’s [Collaborations Workshop 2023 (CW23)](https://www.software.ac.uk/cw23), and in particular on the “Raspberry” discussion and speed blogging session on “How do you track the environmental impact of computing?”. We thank all participants of CW23 for engaging around this topic.  

```